### DISCUSSION TITLE
Can Automa be used with Jenkins or other CI/CD tools for automated testing?

### DISCUSSION THREAD

**Discussion by seasoncool** [AutomaApp/automa#1210] [Q&A]
We want to integrate Automa scripts into our CI/CD workflow and trigger them from other jobs. 
Is this currently possible? Thanks in advance.

**Comment by flashvnn** [comment]
Automa can used with Jenkins or other CI/CD, it required you open chome/firefox that install automa and run automa script.
But for automated testing with Jenkins or other CI/CD i think you should looking cypress: https://www.cypress.io/
